core/base/module.php

```diff
@@ -280,6 +280,20 @@ final protected function get_css_assets_url( $file_name, $relative_url = null, $
 		return $this->get_assets_url( $file_name, 'css', $relative_url, $add_min_suffix );
 	}
 
+	final protected function enqueue_module_assets() {
+		add_action( 'elementor/common/after_register_scripts', function () {
+			wp_enqueue_script(
+				$this->get_name(),
+				$this->get_js_assets_url( $this->get_name() ),
+				[
+					'elementor-common',
```

Comment:
The `common` is available only for high capabilities users, why it depends on it?

Reply:
Handled.